apple/testing/default_runner/macos_test_runner.template.sh

```diff
@@ -156,14 +162,14 @@ llvm_cov_export_status=0
 lcov_args=(
   -instr-profile "$profdata"
   -ignore-filename-regex='.*external/.+'
-  -path-equivalence="$ROOT",.
+  -path-equivalence=.,"$PWD"
```

Comment:
what's the gist of this change again? even when i did this originally i wasn't sure how this would affect things. are ROOT and PWD actually different?

Reply:
hrm so in that case wouldn't `ROOT` be the one in the binaries? it looks to me like `PWD` is really specific to the test 🤔

Reply:
Its not matching up, at least when I run this:

`bazel coverage //examples/ios/HelloWorldSwift:UnitTests --action_env=LCOV_MERGER=/usr/bin/true --experimental_use_llvm_covmap --test_output=all --nocache_test_results --features=swift.coverage_prefix_map`

If I change it back to:

```terminal
  -instr-profile "$profdata"
  -ignore-filename-regex='.*external/.+'
  -path-equivalence="$ROOT",.
)
```

I get misses on all paths

Reply:
What if we pass a label_list instead of a single file. This label list is the list of files to perform filtering with, we could then try to find out how to get those to have the right path relative to `$ROOT`?

Reply:
as long as something like this works for all cases i don't think we need to change what you have here, im just surprised.

Reply:
What else should I test with in this flow? Given that coverage-prefix will be the default and that this works is that sufficient to merge?